src/EventSubscriber.php

```diff
@@ -33,11 +33,9 @@ public function handleEvent(string $eventName, $payload)
 
     public function storeEvent(ShouldBeStored $event)
     {
-        $storedEvent = $this->storedEventModelClass::createForEvent($event);
-
-        $this->eventProjectionist
-            ->callEventHandlers($this->eventProjectionist->projectors, $storedEvent)
-            ->callEventHandlers($this->eventProjectionist->reactors, $storedEvent);
+        $this->eventProjectionist->handle(
```

Comment:
Nice! ❤️

Reply:
@freekmurze correct me if I'm wrong, but isn't the eventProjectionist class -at least in concept- equal to the Illuminate\Events\Dispatcher class? Is it safe to say that Laravel somewhat already follows the Event sourcing pattern only that it does not store the events (except when they are properties of queable jobs) and does not provide the medium for re-executing the stored events? Thank you in advance.